tools/pv_tree_generator/build_tree_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest.mock import patch
+import json
+from collections import defaultdict
+
+import build_tree
+from util import _read_pop_obs_spec, _read_stat_var
+
+class BuildTreeTest(unittest.TestCase):
+    """testing build_tree"""
+
+    @staticmethod
+    def get_sv():
+        """pick a subset of stat_var dcids"""
+        level0 = ['MarriedPopulation', 'DivorcedPopulation', 'MalePopulation']
+        level1 = ['dc/0k7719speyv21', 'dc/2pvw6jqmkp41b', 'dc/06f6zh0wslnx',\
+               'dc/2rjldly6tsmf', 'dc/026gmdj2xk1kb', 'dc/f7g49v7tzy3rd']
+        level2 = ['dc/61fzldryrnte1', 'dc/6yb4mgxtc1288', 'dc/esr27kls5vfy6',\
+                'dc/jhpflf91nvlt6', 'dc/cmn38d85glq72', 'dc/pj50xtdgxeh4g',\
+                'dc/pvsbze841l2tc', 'dc/tpeg0jxdts2t3']
+        sv_dcid = level0 + level1 + level2
+        return sv_dcid
+
+    @staticmethod
+    def get_triples_(dcids):
+        """read the triples with predicates in the specified list"""
+        triples = json.load(open("test_triples.json", "r"))
+        results = defaultdict(list)
+        #skip if the predicate is not in the list
+        predicates = ["measuredProperty", "populationType", "statType", "income", \
```

Comment:
suggest not to use "\" for line continuation. http://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
got it.